### PR TITLE
Add default  maven goal.

### PR DESCRIPTION
**Pull request by arturobernalg**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -462,6 +462,7 @@
   </dependencyManagement>
 
   <build>
+    <defaultGoal>clean verify apache-rat:check javadoc:javadoc</defaultGoal>
```
Comment (juanpablo-santos) [inline pom.xml:465]: Hi,

I'd rather not generate the javadocs as part of the default build, doesn't make much sense to me. Also, on the build cheatsheet we're stating they're not generated, WDYT?

**Review by juanpablo-santos — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (arturobernalg) [reply pom.xml]: changed.

**Review by juanpablo-santos — APPROVED** [review]
lgtm

**Comment by juanpablo-santos** [comment]
merged in 2.12.0-git-06, thanks!

**Pull request merged**